Formula/ipinfo-cli.rb

```diff
@@ -0,0 +1,21 @@
+class IpinfoCli < Formula
+  desc "Official CLI for the IPinfo IP Address API"
+  homepage "https://ipinfo.io/"
+  url "https://github.com/ipinfo/cli/archive/ipinfo-1.1.2.tar.gz"
+  sha256 "4172c686b15d1a92f39b3490eea4f56325d7b5f9e6696c1d12c4aedc9706a2d6"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  conflicts_with "ipinfo",
+    because: "homebrew/core/ipinfo conflicts with the binary name provided by this formula"
+
+  def install
+    system "./ipinfo/build.sh"
```

Comment:
Does this download anything?

Reply:
No, it only runs the `go` toolchain.

Reply:
One thing to clarify: all dependencies are vendored, so the `go` toolchain isn't pulling anything either.